conda_environment.yml

```diff
@@ -0,0 +1,25 @@
+name: geocat-viz
+channels:
+  - conda-forge
+  - ncar
+dependencies:
```

Comment:
In general, such conda_environment files are aimed at "building from source" (for those who won't install via conda, and instead build from source and maybe contribute further with their own development. If we have this file, then:

- Several dependencies listed in it seem like not needed for building/testing purposes (e.g. `metpy`, `sphinx`, etc). 
- Shouldn't we add a "Building from source" part into INSTALLATION.md?

Reply:
Good point! Added